RELEASING.md

```diff
@@ -1,13 +1,16 @@
-1. Start a branch `release/x.x.x`
+### Releasing a version:
+
+1. Create a `fastlane/.env` file with your GitHub API token (see `fastlane/.env.SAMPLE`). This will be used to create the PR, so you should use your own token so the PR gets assigned to you.
+1. Run `bundle exec fastlane bump`
+    1. Input new version number
+    2. Update CHANGELOG.latest.md to include the latest changes. Call out API changes (if any). You can use the existing CHANGELOG.md as a base for formatting. To compile the changelog, you can compare the changes between the base branch for the release (usually main) against the latest release, by checking https://github.com/revenuecat/purchases-flutter/compare/<latest_release>...<base_branch>. For example, https://github.com/revenuecat/purchases-flutter/compare/3.10.0...main. 
+    3. A new branch and PR will automatically be created
 1. Update to the latest SDK versions in ios/purchases_flutter.podspec, macos/purchases_flutter.podspec and android/build.gradle.
-1. Run `flutter format .`
-1. Update versions in VERSIONS.md.
-1. Update CHANGELOG-LATEST.md with the changes for the current version (to be used by Fastlane for the github release notes)
-1. Run `bundle exec fastlane bump_and_update_changelog version:X.Y.Z` (where X.Y.Z is the new version) to update the version number in pubspec.yaml, purchases_flutter.podspec and android/build.gradle.
-1. `flutter pub pub publish --dry-run`
+1. Update versions in VERSIONS.md and push changes
 1. If purchases-hybrid-common was updated, run `pod update PurchasesHybridCommon` in both `MagicWeather` and `purchase_tester`
-1. Commit the changes, open a PR into main, merge when ready
+1. `flutter pub pub publish --dry-run`
+1. Wait until PR approved and make sure local copy matches remote
 1. `git tag x.y.z`
-1. `git push origin main && git push --tags`
+1. `git push origin --tags`
 1. Create a new release in github
 1. `flutter pub pub publish`
```

Comment:
Much of this process can still be automated but I just moved to use the `bump` action in the fastlane plugin for now. I will add the rest in future PRs